src/app/shared/components/message-bar/message-bar.component.html

```diff
@@ -10,7 +10,17 @@
       </p>
     </div>
   </span>
-  <div *ngIf="!data.unclosable" (click)="onClose()" data-testid="closing-button">
-    <mat-icon class="action">close</mat-icon>
+  <div matSnackBarActions>
+    <div
+      matSnackBarAction
+      *ngIf="!data.unclosable"
+      (click)="onClose()"
+      (keydown)="onKeyDown($event)"
+      tabindex="0"
+      data-testid="closing-button"
+      class="action"
+    >
```

Comment:
Put the `>`  at the end of the last line instead of being alone on the next line

Reply:
Done